src/flow/src/adapter.rs

```diff
@@ -97,6 +98,7 @@ impl Default for FlownodeOptions {
             cluster_id: None,
             node_id: None,
             grpc: GrpcOptions::default().with_addr("127.0.0.1:3004"),
+            frontend_addr: None,
```

Comment:
question: what if there are multiple frontend instances? Should we use a list of frontend addr?

Reply:
> question: what if there are multiple frontend instances? Should we use a list of frontend addr? Or it's ok to use any of the frontend instances, which means we can specify it as a load balancer addr.

Yes as long as they are grpc service